src/diffusers/pipelines/audioldm2/modeling_audioldm2.py

```diff
@@ -145,6 +145,21 @@ def forward(
         )
 
 
+class AudioLDMLearnablePositionalEmbedding(ModelMixin, ConfigMixin):
```

Comment:
Would this be cleaner as an attribute of the `AudioLDM2ProjectionModel`? We could add a new argument `learned_positional_embeddings=False`. If `learned_positional_embeddings` is False, then we do the current behaviour. If `learned_positional_embeddings` is True, then we add the learned embeddings to the VITS text encoder hidden-states

IMO this is cleaner than adding a new model, since we don't have to redefine args like `hidden_size_2`. It should also help with using a single pipeline for both tasks, since we only need to import `AudioLDM2ProjectionModel` in the main pipeline

Reply:
Great!

Reply:
Is this object used anywhere? If not can it be removed?